src/PageTransition.js

```diff
@@ -184,11 +187,18 @@ class PageTransition extends React.Component {
   }
 
   onEnter() {
+    const { monkeyPatchScrolling } = this.props
+
     // It's safe to reenable scrolling now
     this.disableScrolling = false
     this.setState({
       state: 'enter',
       showLoading: false,
+    }, () => {
```

Comment:
Apologies for not having CI set up on this repo to warn about this automatically, but this needs to be run through prettier. `npm run prettier` should get everything in good shape.